### PR TITLE
Add WNP for Firefox 134beta [fix #15489]

### LINKED ISSUE #15489: WNP 134 Beta (English)
## Description

The community team reached out and asked for support in getting a WNP to English users on 134 Beta.

This project does not have design or copy-writing support, so we are working with limited resources.

Copy: [Link](https://docs.google.com/document/d/1vyyJdhiNUY92IKKgU2WOqzn0rVro0G6EIgGLVrsR9lg/edit?usp=sharing)

Design:

Recommendation is a very lightweight design that is the following from top to bottom:

- Firefox logo
- Headline
- Body Copy
- CTA Link

We would use our typical wrapper, which includes WNP header, footer, and mofo donation link.

Below is a screenshot of a simple layout that this can follow

---

image

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds a what's new page for Firefox 134beta. English only.

## Issue / Bugzilla link
#15489 

## Testing
http://localhost:8000/en-US/firefox/134.0beta/whatsnew/

### CODE REVIEW

**Comment by craigcook** [comment]
Marking Do Not Merge because we're waiting for legal sign off on the copy. Also the CTA links to a 404, as that campaign page needs to be created.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx134beta.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx134beta.html:37]: This page needs to be created, and we may need to update the URL if the final URL differs.

**Review by alexgibson — APPROVED** [review]
r+wc to merge when we get the final link

File: media/js/firefox/whatsnew/whatsnew-134beta.js
Comment (alexgibson) [inline media/js/firefox/whatsnew/whatsnew-134beta.js:1]: I added a `firefox_whatsnew_events` bundle in https://github.com/mozilla/bedrock/pull/15512 that can be re-used between pages rather than repeating this code every WNP

**Comment by craigcook** [comment]
We have legal signoff and assurances the community page will go live before release day.

**Pull request merged**
